src/SprykerEco/Service/ComputopApi/ComputopApiServiceInterface.php

```diff
@@ -134,6 +135,7 @@ public function getBlowfishDecryptedValue($cipherText, $length, $password);
     /**
      * Specification:
      * - Generate ReqId from QuoteTransfer for Computop calls
+     * - Requires `QuoteTransfer.TotalsTransfer.hash` and `QuoteTransfer.CustomerTransfer`
```

Comment:
```suggestion
     * - Requires `QuoteTransfer.totals.hash` and `QuoteTransfer.customer` properties to be set.
```

Reply:
please make it 1st line of specification
please also make sure that properties you described are really required to avoid situations as per method above
same for similar cases below

Reply:
Fixed